cluster-api/src/main/java/com/sequenceiq/cloudbreak/cluster/service/ClusterComponentConfigProvider.java

```diff
@@ -51,6 +58,26 @@ public ClusterComponentView getComponentView(Long clusterId, ComponentType compo
         return componentViewRepository.findOneByClusterIdAndComponentTypeAndName(clusterId, componentType, name);
     }
 
+    public ClouderaManagerRepo getClouderaManagerRepoDetails(Long clusterId) {
+        try {
+            ClusterComponent component = getComponent(clusterId, ComponentType.CM_REPO_DETAILS);
+            return component == null ? null : component.getAttributes().get(ClouderaManagerRepo.class);
```

Comment:
`ClusterComponent.attributes` is nullable inthe entity, so it should occure NPE here.

Reply:
this has been the case for all other usage in this class also, but I fixed those too